jujugui/static/gui/src/app/components/inspector-config/inspector-config.js

```diff
@@ -109,11 +109,11 @@ YUI.add('inspector-config', function() {
     },
 
     /**
-      Handle cancelling the changes and returning to the inspector overview.
+      Return to the inspector overview.
 
-      @method _handleCancelChanges
+      @method _showInspectorIndex
     */
-    _handleCancelChanges: function() {
+    _showInspectorIndex: function() {
```

Comment:
Elsewhere we call this the service overview, you could use that here too if you liked.